src/app/modules/courses/components/search/search.component.html

```diff
@@ -1,6 +1,8 @@
-<form (submit)="submit()">
-  <div class="content">
-    <input name="searchText" type="text" [(ngModel)]="searchText">
-    <button type="submit">Search</button>
-  </div>
-</form>
+<div class="content">
+  <input
+    name="searchText"
+    type="text"
+    [(ngModel)]="searchText"
+    (keyup)="submit()"
```

Comment:
It is up to you, but form UX I would suggest to keep logic with button

Reply:
there were a task to remove button